setup.cfg

```diff
@@ -1,7 +1,7 @@
 [metadata]
 name = chromo
 # version = attr: chromo.__version__
-version = 0.3.0rc2
+version = 0.4.0rc1
```

Comment:
How do you want to proceed after this is merged? You modify this on the main branch to promote to real pypi?

Reply:
On real PyPI the version would be 0.4.0. I wanted to submit it to Test PyPI, but build for python3.8 on Linux doesn't pass tests because latest `particle` is not available for python3.8.